libdnf5/base/base_impl.hpp

```diff
@@ -77,6 +80,18 @@ class Base::Impl {
     libdnf5::advisory::AdvisorySack rpm_advisory_sack;
 
     plugin::Plugins plugins;
+    LogRouter log_router;
+    ConfigMain config;
+    repo::RepoSack repo_sack;
+    rpm::PackageSack rpm_package_sack;
+    module::ModuleSack module_sack;
+    std::map<std::string, std::string> variables;
+    transaction::TransactionHistory transaction_history;
+    Vars vars;
+    std::unique_ptr<repo::DownloadCallbacks> download_callbacks;
+
+    WeakPtrGuard<LogRouter, false> log_router_gurad;
+    WeakPtrGuard<Vars, false> vars_gurad;
```

Comment:
I know it was already there, but when we're already touching this part of code, please fix these typos:
- log_router_gurad -> log_router_guard
- vars_gurad -> vars_guard

Reply:
Nice catch, fixed.